services_app/src/main/java/org/opendatakit/services/sync/service/logic/ProcessManifestContentAndFileChanges.java

```diff
@@ -961,6 +972,10 @@ public boolean syncRowLevelFileAttachments(String serverInstanceFileUri, String
 
     if ( fullySyncedUploads && fullySyncedDownloads ) {
       log.i(LOGTAG, "syncRowLevelFileAttachments SUCCESS syncing file attachments for " + instanceId);
+      if ( attachmentState == SyncAttachmentState.REDUCED_DOWNLOAD || attachmentState == SyncAttachmentState.SYNC_WITH_REDUCED_DOWNLOAD && atleast1Download ) {
```

Comment:
Not sure that we want to return false here, wouldn't a full sync down with reduced file sizes be complete?

Reply:
Removed this and changed it to using a Properties field as we discussed.